pkg/asset/agent/image/agentimage.go

```diff
@@ -160,10 +192,28 @@ func (a *AgentImage) PersistToFile(directory string) error {
 	// Remove symlink if it exists
 	os.Remove(agentIsoFile)
 
-	err := isoeditor.Create(agentIsoFile, a.tmpPath, a.volumeID)
-	if err != nil {
-		return err
+	var err error
+	// For external platform when the bootArtifactsBaseUrl is specified,
+	// output the rootfs file alongside the minimal ISO
+	if a.platform == external.Name && a.bootArtifactsBaseURL != "" {
```

Comment:
I would suggest adding some user facing documentation explaining how to use the external platform. Is bootArtifactsBaseURL required to be set if platform is external? Should there be validation if it is required?

The documentation should explain that if bootArtifactsBaseURL is set a minimal ISO is generated with the rootFS in X directory.